src/llm/agent-evals/slack-agent/test-data.ts

```diff
@@ -0,0 +1,270 @@
+import { ToolResponseTypeMap } from './mock';
+import { TestCase } from '../common/types';
+
+export const testCases: TestCase<ToolResponseTypeMap>[] = [
+  {
+    description:
+      'List all public channels in the Slack workspace after a discussion about onboarding.',
+    chat_history: [
+      { author: 'Alice_Hightower', message: 'We need to onboard new members soon.' },
+      { author: 'Slack_Bot', message: "Let's make sure they know all the channels." }
```

Comment:
This message can be from some random user, using slack_bot here makes it confusing
```suggestion
      { author: 'Joh_Doe', message: "Let's make sure they know all the channels." }
```